### ISSUE TITLE
The main website `https://twitter.com/tjcchen?failedScript=vendor` is down, please help to take a look?

### ISSUE THREAD

**Issue by tjcchen** [twitter/opensource-website#199]
Hi there,

The company website is down, could someone help to take a look.

time: 2023/09/22 17:40
url: https://twitter.com/tjcchen?failedScript=vendor

screenshots:
<img width="1433" alt="Screen Shot 2023-09-22 at 5 45 20 PM" src="https://github.com/twitter/opensource-website/assets/6133656/f26c554a-3702-4597-9588-25d1efa5feda">

<img width="1432" alt="Screen Shot 2023-09-22 at 5 45 58 PM" src="https://github.com/twitter/opensource-website/assets/6133656/b81a865c-b3ce-41cf-9572-e54bf490cbf1">

Retried to refresh the page multiple times, still doesn't work. 

Thanks in advance!

Regards,
Andy

**Comment by vaibhavdhotre123** [comment]
I tried to open This site and it opens may be your wifi firewall didn't allow site to open, In my case It open Andy profile. else  you can clear cookies of browser and run it, It will work .

**Comment by tjcchen** [comment]
Yes, I guess it was just temporarily down long time ago. Closing this ticket.
